public/app/plugins/panel/dashlist/styles.ts

```diff
@@ -2,7 +2,15 @@ import { css } from '@emotion/css';
 
 import { GrafanaTheme2 } from '@grafana/data';
 
+import { alpha } from '../../../../../packages/grafana-data/src/themes/colorManipulator';
```

Comment:
You can import `colorManipulator` from `@grafana/data` and use `alpha` method from it.